### DISCUSSION TITLE
Tabs fade out under left/right buttons when extending the display width

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#16572] [General - Components / Directives / etc] [answered]
I am still on Quasar v1, not sure whether this affects Quasar v2, but when tabs extend the width, the left/right arrows appear above the text and the are overlaid, therefore the UI becomes a bit confusing. 

Is it possible to fade out the tabs under the arrows?

**Comment by rstoenescu** [comment, marked as answer]
You can use the `outside-arrows` Boolean prop.
Example from docs: https://v1.quasar.dev/vue-components/tabs#example--outside-inside-and-visible-on-mobile-arrows

The fade out cannot be achieved safely from the framework level. You'll need a bit of CSS for it. Shouldn't be too hard.

**Comment by i-mage** [comment]
Thank you!
